django/db/migrations/writer.py

```diff
@@ -503,6 +509,8 @@ def serialize(cls, value):
 
 MIGRATION_TEMPLATE = """\
 # -*- coding: utf-8 -*-
+# Generated by Django %(version)s
+# on %(timestamp)s
```

Comment:
@MarkusH and I discussed that, we decided keep the timestamp on a separate line to the version string to keep things clearer where django.get_version() returns large strings which include the date such as '1.9.dev20150712174416'.